raft/test/raft_test.cpp

```diff
@@ -575,8 +574,8 @@ namespace bzn
 
```

Comment:
I think commitindex should be 1 on this first request

Reply:
Thanks for your help Isabel.